lte/gateway/c/core/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -1721,15 +1721,19 @@ void mme_app_handle_release_access_bearers_resp(
     OAILOG_FUNC_OUT(LOG_MME_APP);
   }
 
+  ue_context_p->nb_rabs--;
+  /* Wait for all the RAB responses, in case RAB req was sent to both
+   * spgw and s8 tasks
+   */
+  if (ue_context_p->nb_rabs) {
+    OAILOG_FUNC_OUT(LOG_MME_APP);
+  }
```

Comment:
what about the s1u_bearer stats? what happens if one of the tasks fail to respond.

Reply:
Since MME will not know which PDN sessions are mapped to which task it cannot update the stats only when one task returns a rsp msg. It has to wait for both responses. There is no spec defined timer to guard these requests. Even in the earlier code MME would wait for all the responses and then decrement stats

Reply:
Of course we know which PDN session is mapped to which task; you use that knowledge to route the requests to s8 or  spgw.